app/src/main/java/com/example/mytrainingpal/components/EnterPainPrompt.kt

```diff
@@ -0,0 +1,57 @@
+package com.example.mytrainingpal.components
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Edit
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+@Composable
+fun EnterPainPrompt(navigateToMusclePain: () -> Unit) {
+
+// Card Reference: https://foso.github.io/Jetpack-Compose-Playground/material/card/
+    Card(
+        border = BorderStroke(2.dp, MaterialTheme.colors.secondary),
+        backgroundColor = MaterialTheme.colors.primary,
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(10.dp),
+        elevation = 10.dp,
+
+    ){
+        Column(
+            //modifier = Modifier.padding(20.dp)
```

Comment:
If this is not needed anymore, please delete this line 😄

Reply:
done.